app/frontend/websocket/handlers/symbol-setting-delete.js

```diff
@@ -11,6 +11,8 @@ const handleSymbolSettingDelete = async (logger, ws, payload) => {
 
   const { symbol } = symbolInfo;
 
+  await queue.hold(logger, symbol);
```

Comment:
@uhliksk 

I think we can refactor this pattern
- queue.hold
- some action
- queue.executeFor

Something like this 
![image](https://user-images.githubusercontent.com/5715919/210363504-f1e88b2a-225d-447c-8353-a607c1687144.png)

If you agree, I will refactor the code.